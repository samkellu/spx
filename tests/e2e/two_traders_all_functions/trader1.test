[Trader 0] Received from SPX: MARKET OPEN;
[Trader 0] Event: SELL 0 GPU 30 500;
[Trader 0] Received from SPX: ACCEPTED 0;
[Trader 0] Event: SELL 1 GPU 30 501;
[Trader 0] Received from SPX: ACCEPTED 1;
[Trader 0] Event: SELL 2 GPU 30 501;
[Trader 0] Received from SPX: ACCEPTED 2;
[Trader 0] Event: SELL 3 GPU 30 502;
[Trader 0] Received from SPX: MARKET BUY GPU 30 499;ACCEPTED 3;
[Trader 0] Event: SELL 4 Router 22 51;
[Trader 0] Received from SPX: ACCEPTED 4;
[Trader 0] Event: CANCEL 0;
[Trader 0] Received fr