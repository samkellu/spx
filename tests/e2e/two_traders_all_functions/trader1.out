[Trader 0] Received from SPX: MARKET OPEN;
[Trader 0] Event: SELL 0 GPU 30 500;
[Trader 0] Received from SPX: ACCEPTED 0;
[Trader 0] Received from SPX: MARKET BUY GPU 30 499;
[Trader 0] Event: SELL 1 GPU 30 501;
[Trader 0] Received from SPX: ACCEPTED 1;
[Trader 0] Received from SPX: MARKET BUY GPU 30 500;
[Trader 0] Event: SELL 2 GPU 30 501;
[Trader 0] Received from SPX: FILL 0 30;
[Trader 0] Received from SPX: ACCEPTED 2;
[Trader 0] Received from SPX: MARKET BUY GPU 30 500;
[Trader 0] Event: SELL 3 GPU 30 502;
[Trader 0] Received from SPX: ACCEPTED 3;
[Trader 0] Received from SPX: MARKET BUY GPU 30 501;
[Trader 0] Event: SELL 4 Router 22 51;
[Trader 0] Received from SPX: FILL 1 30;
[Trader 0] Received from SPX: FILL 2 0;
[Trader 0] Received from SPX: ACCEPTED 4;
[Trader 0] Received from SPX: MARKET BUY Router 22 50;
[Trader 0] Event: CANCEL 0;
[Trader 0] Received from SPX: INVALID;
[Trader 0] Event: AMEND 1 30 502;
[Trader 0] Received from SPX: INVALID;
[Trader 0] Event: AMEND 2 30 503;
[Trader 0] Received from SPX: AMENDED 2;
[Trader 0] Received from SPX: MARKET BUY GPU 30 502;
[Trader 0] Event: AMEND 3 30 504;
[Trader 0] Received from SPX: FILL 3 30;
[Trader 0] Received from SPX: INVALID;
[Trader 0] Event: AMEND 4 22 505;
[Trader 0] Received from SPX: AMENDED 4;
[Trader 0] Received from SPX: MARKET BUY Router 22 504;
[Trader 0] Event: SELL 5 Router 100000 100000;
[Trader 0] Received from SPX: ACCEPTED 5;
